python_scripts/cross_validation_ex_04.py

```diff
@@ -13,7 +13,7 @@
 
 # %% [markdown]
 # The first step is to create a model. Use a machine learning pipeline
-# composed of a scaler followed by a logistic regression classifier.
+# composed of a scaler followed by a logistic regression classification model.
```

Comment:
Here too either the class name or just "logistic regression"

logistic regression classification model is super weird ...